spec/model/post_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before do
+    @user = User.create(name: 'Halle', photo: 'Photo', bio: 'Ruby Developper', posts_counter: 0)
+    @post = Post.create(user: @user, title: 'Hello', text: 'Hello Rails', comments_counter: 0, likes_counter: 0)
+    @comments = []
+  end
+
+  it 'Should be present and <= 250 characters' do
+    @post.title = 'Hello'
+    expect(@post).to be_valid
+  end
+
+  it 'comments_counter should be an integer and sould be  >= 0 ' do
+    @post.comments_counter = 0
+    expect(@post).to be_valid
+  end
+
+  it 'likes_counter should be an integer and sould be  >= 0' do
+    @post.likes_counter = 0
+    expect(@post).to be_valid
+  end
```

Comment:
- [OPTIONAL] Good job so far with the test cases, but since we are aiming for a 100% test coverage, it will be nice to also include negative test cases, that is tests that should evaluate to false, or not valid.